src/js/core/widget/core/PanelChanger.js

```diff
@@ -83,6 +84,28 @@
 			PanelChanger.classes = classes;
 			PanelChanger.prototype = prototype;
 
+			function writeToCookie(storageName, value) {
```

Comment:
I think this is function which can be useful in future. How about moving it to utils?

Reply:
I'll move these methods to `tau.util.cookie`